### PR TITLE
ICMSLST-1599 Add confirm message for close case action.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

**Comment by MattHolmes123** [comment]
Popup when pressing Stop Case:
image

**Comment by MattHolmes123** [comment]
Success message after closing:
image

**Review by mikeylockwood — APPROVED** [review]

File: web/domains/case/views/views_misc.py [hunk 1]
```diff
@@ -352,6 +352,12 @@ def manage_case(
 
                     send_email(subject, body, recipients)
 
+                messages.success(
+                    request,
+                    "This case has been stopped and removed from your workbasket."
+                    " It will still be available from the search screen.",
+                )
+
                 return redirect(reverse("workbasket"))
```
Comment (davidwtbuxton) [inline web/domains/case/views/views_misc.py:361]: Not part of this change, I know, but one can use `redirect("workbasket")` directly https://docs.djangoproject.com/en/4.0/topics/http/shortcuts/#redirect

**Review by davidwtbuxton — APPROVED** [review]

File: web/domains/case/views/views_misc.py (hunk 1 above)
Comment (MattHolmes123) [reply web/domains/case/views/views_misc.py:361]: :+1:
